takeshape/penny
# Switch from recharge to native Shopify queries

### Test Plan (Steps to test):

1. Check the Mesh Pants product to see if it still has a subscription option
1. Add it to your cart, do you see it as a subscription item?
